config/config.go

```diff
@@ -902,17 +906,20 @@ func escapeEnv(value string) string {
 }
 
 func loadConfig(config string) ([]byte, error) {
-	u, err := url.Parse(config)
-	if err != nil {
-		return nil, err
-	}
+	if fetchURLRe.MatchString(config) {
+		u, err := url.Parse(config)
+		if err != nil {
+			return nil, err
+		}
 
-	switch u.Scheme {
-	case "https", "http":
-		return fetchConfig(u)
-	default:
-		// If it isn't a https scheme, try it as a file.
+		switch u.Scheme {
+		case "https", "http":
+			return fetchConfig(u)
+		default:
+			// If it isn't a https scheme, try it as a file
```

Comment:
You can leave this one out as it does nothing and move the comment next to the last return. Just a minor comment though.

Reply:
I think the linter will complain then?